server/Todo.ts

```diff
@@ -0,0 +1,51 @@
+import { Sequelize,Model,DataType,CreationOptional,InferAttributes,InferCreationAttributes, DataTypes} from 'sequelize';
+import * as dotenv from 'dotenv'
+dotenv.config()
```

Comment:
You have config.js utility which is good practise for using dotenv. Why you're not using it?